### PR TITLE
feat:Banner UI

### PR DESCRIPTION
**Pull request by IzaDoliashvili**
# Pull Request Template for ArtOa

## Description

Please provide a summary of the changes in this pull request:

- What does this PR do?
- Why are these changes necessary?
- Any relevant links or references?

Fixes: [issue number/link]

---

## Type of Change

Please select the type of change this PR introduces (tick the appropriate box):

- [ ] 🐛 Bug fix (non-breaking change that fixes an issue)
- [ ] ✨ New feature (non-breaking change that adds functionality)
- [ ] 🚨 Breaking change (fix or feature that would cause existing functionality
      to not work as expected)
- [ ] 🔄 Refactor (code or file structure changes without affecting functionality)
- [ ] 📝 Documentation update (changes or additions to documentation)
- [ ] 🚀 Performance improvement (optimization in code or resources)

---

## Checklist

Please ensure the following tasks have been completed before requesting a review:

- [ ] My code follows the style guidelines of this project
- [ ] I have run tests to ensure my changes work as expected
- [ ] I have added/updated necessary tests or documentation (if applicable)
- [ ] I have self-reviewed my code and refactored any areas that need improvement
- [ ] There are no console errors or warnings related to my changes
- [ ] The PR includes a clear, detailed title and description
- [ ] Any relevant issues have been linked or mentioned in this PR
- [ ] I have reviewed the acceptance criteria and met all requirements
- [ ] I have considered edge cases and errors that may occur due to my changes

---

## Testing Instructions

Please describe how your changes can be tested:

1. **Test Setup:** (Any steps for environment setup)
2. **Test Steps:**
   - Step 1: ...
   - Step 2: ...
   - ...

---

## Screenshots / Visual Changes

Please attach any relevant screenshots, videos, or GIFs to illustrate the impact
of this change (if applicable).

---

## Additional Notes

Is there anything else the reviewer should know about this PR? Any known issues,
challenges, or decisions made during implementation?

### CODE REVIEW

File: src/app/banner/page.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { Button } from "@/components/ui/button";
+
+export default function Banner() {
+  return (
+    <section className="relative bg-cover bg-center text-white">
```
Comment (mshubitidze) [inline src/app/banner/page.tsx:5]: text-white is unnecessary here, text color is already being set on all elements in globals.css file

File: src/app/banner/page.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      <div
+        className="w-full"
+        style={{ backgroundImage: "url('/banner/image/Mask.png')" }}
```
Comment (mshubitidze) [inline src/app/banner/page.tsx:8]: this is completely fine, although i'd inline this style as a tailwind class: `bg-[url('/banner/image/Mask.png')]`

File: src/app/banner/page.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      >
+        <div className="bg-black/70 px-8 py-16">
```
Comment (mshubitidze) [inline src/app/banner/page.tsx:10]: background mask is already present in the image uploaded, adding bg-black/70 is unnecessary here
Screenshot 2024-11-24 at 12 05 57

File: src/app/banner/page.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+            <div>
+              <Button> Join As </Button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div className="bg-secondary py-8 text-gray-300">
+        <div className="mx-10 flex max-w-5xl justify-around">
+          <div className="text-center">
+            <p className="text-2xl font-bold md:text-3xl">70+</p>
```
Comment (mshubitidze) [inline src/app/banner/page.tsx:35]: font sizes and the layout dont look right
Screenshot 2024-11-24 at 12 05 57

File: src/app/banner/page.tsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+        className="w-full"
+        style={{ backgroundImage: "url('/banner/image/Mask.png')" }}
+      >
+        <div className="bg-black/70 px-8 py-16">
+          <div className="mx-10 flex max-w-2xl flex-col">
+            <h1 className="text-4xl font-bold md:text-5xl">
+              Discover The Imaginary World
+            </h1>
+            <div className="mx-auto mt-4 h-1 w-full bg-yellow-500"></div>
+            <div className="my-6 flex flex-col items-start gap-y-1 text-lg leading-relaxed md:text-xl">
```
Comment (mshubitidze) [inline src/app/banner/page.tsx:16]: this layout is close but can me adjusted to be much more similar to the figma design

**Review by mshubitidze — CHANGES_REQUESTED** [review]
very clean code, although ui doesnt match the figma design

**Comment by IzaDoliashvili** [comment]
I have some problems with the background image on mobile version. Please check it.

**Comment by mshubitidze** [comment]
> I have some problems with the background image on mobile version. Please check it.

i think its because the div with bg-[url(image)] class doesnt have bg-cover class
